estate/models/estate_property.py

```diff
@@ -0,0 +1,109 @@
+from odoo import fields, models, api
+from odoo.exceptions import UserError, ValidationError
+from odoo.tools import float_utils
+
+
+
+class Property(models.Model):
+    _name = "estate.property"
+    _description = "Real Estate Application"
+    _sql_constraints = [
```

Comment:
```suggestion
    _description = "Real Estate Application"
    
    _sql_constraints = [
```
sorry for being a bit annoying with spaces and indentation, but the `_name` and `_description` are usually separated with one space from the rest